app/src/main/kotlin/at/bitfire/davdroid/repository/DavCollectionRepository.kt

```diff
@@ -343,9 +347,16 @@ class DavCollectionRepository @Inject constructor(
                                 text(DavUtils.ARGBtoCalDAVColor(it))
                             }
                         }
-                        timezoneDef?.let {
-                            insertTag(CalendarTimezone.NAME) {
-                                cdsect(it)
+                        timezoneDef?.let { timezoneId ->
```

Comment:
I'd expect `Def` to be the full definition, while `Id` only the ID. So it can't be the same.